CODE_OF_CONDUCT.md

```diff
@@ -1,113 +1,11 @@
-This document was copied from its home at https://publiclab.org/conduct on October 12, 2017. See that page if you would like to submit your concerns in a safe, completely anonymous way, and to learn more about this document. 
-
-****
-
 # Public Lab Code of Conduct 
 
 _Public Lab, PO Box 426113, Cambridge, MA 02142_
```

Comment:
Hey @jywarren isn't this address outdated one?

Reply:
I see Providence address - Public Lab, 55 Cromwell Street, 1C, Providence, RI 02907 at publiclab.org/conduct. So, should I update the address or remove it?

Thanks!